docs/source/_templates/sdk_overview_tutorial.rst

```diff
@@ -53,17 +53,42 @@ to create transactions and submit them as :term:`Sawtooth batches<Batch>`.
 
 {% endif %}
 
-
 .. note::
 
    This tutorial demonstrates the relevant concepts for a Sawtooth transaction
    processor and client, but does not create a complete implementation.
 
-   * For a full implementation of the tic-tac-toe transaction family, see
-     ``/project/sawtooth-core/sdk/examples/xo_{{ lowercase_lang }}/``.
 
-   * For full implementations in other languages, see
-     ``https://github.com/hyperledger/sawtooth-core/tree/master/sdk/examples``.
+{% if language == 'Rust' %}
+
+   For a full Rust implementation of the XO transaction family, see
+   ``/{project}/sawtooth-core/sdk/examples/xo_{{ lowercase_lang }}/``.
```

Comment:
This link does not expand in the generated HTML

Reply:
(ditto)